cmd/copy.go

```diff
@@ -354,6 +355,11 @@ func (raw rawCopyCmdArgs) cook() (CookedCopyCmdArgs, error) {
 		return cooked, err
 	}
 
+	cooked.putBlobSize, err = blockSizeInBytes(raw.putBlobSizeMB)
```

Comment:
todo: validate this size is within MaxPutBlobSize

Reply:
Validation is performed in ste. I prefer it that way since that is the centralized location that all commands call into.